inc/dbmysqliterator.class.php

```diff
@@ -194,12 +194,10 @@ function buildQuery ($table, $crit = "", $log = false) {
                      break;
 
                   case 'JOIN' : // deprecated
+                     Toolbox::deprecated('"JOIN" is deprecated, please use "LEFT JOIN" instead.');
```

Comment:
I do not like this way; and as far as I remember this already cause difficult to understand issues.